src/main/kotlin/blackjack/domain/CardSet.kt

```diff
@@ -0,0 +1,25 @@
+package blackjack.domain
+
+import java.util.LinkedList
+
+/**
+ * ### 카드 세트를 표현하는 클래스입니다 기본적으로 52장의 중복없는 카드 모음을 가집니다
+ *
+ * 카드 세트에서 카드를 한장씩 뽑을 수 있습니다
+ */
+class CardSet(
+    cards: Set<Card> = Card.CARD_SET
+) {
+    private val _cards: LinkedList<Card> = LinkedList<Card>()
+        .apply {
+            addAll(cards)
```

Comment:
큰 차이는 없으나 아래처럼도 사용하실 수 있습니다. 메서드 호출이 1개로 줄면 가독성이 조금 올라갈 것 같아요.
```suggestion
    private val _cards: LinkedList<Card> = LinkedList<Card>(cards)
        .apply {
```